### PR TITLE
Update the compilation instructions

### PR DESCRIPTION
**Pull request by cor3ntin**
See https://github.com/cor3ntin/aseba/blob/building_md/docs/en/development/building.rst

### CODE REVIEW

**Comment by cor3ntin** [comment]
@mbonani can we merge that ?

File: docs/en/development/building.rst [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+Getting Started on OSX
+----------------------
+
+You will need OSX 10.11 or greater
+
+-  Install `Homebrew <https://brew.sh/>`__.
+-  In the cloned repository run
+
+   brew update brew tap homebrew/bundle brew bundle
```
Comment (mbonani) [inline docs/en/development/building.rst]: this is a command it should be shown as the others

File: docs/en/development/building.rst [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+~~~~~~~~~~~~~~
+
+In a msys2 prompt, run
+
+::
+
+    mkdir build && cd build
+    cmake -DBUILD_SHARED_LIBS=OFF -DCMAKE_BUILD_TYPE=Release -DCMAKE_PREFIX_PATH="<path of qt>;<path of bonjour>" ..
+    make
+
```
Comment (mbonani) [inline docs/en/development/building.rst]: this implies that the git clone was done inside msys2 no? perhaps better explain that somehow

Comment (cor3ntin) [reply docs/en/development/building.rst]: It does not, you just need to be in the location the repository was clone in. that may be anywhere on your drive

Comment (mbonani) [reply docs/en/development/building.rst]: ok

**Comment by mbonani** [comment]
we can merge when the conflict is solved

**Pull request merged**